packages/flutter_tools/test/src/fake_process_manager.dart

```diff
@@ -123,48 +123,63 @@ class FakeCommand {
   }
 }
 
-class _FakeProcess implements io.Process {
-  _FakeProcess(
-    this._exitCode,
-    Duration duration,
-    this.pid,
-    this._stderr,
+@visibleForTesting
```

Comment:
How about a class doc comment?

Reply:
Good call. Done -- or rather, rephrased the ctor comment to a class doc comment, which is definitely the better place for it.